tox.ini

```diff
@@ -176,7 +188,7 @@ addopts = -rxs
 [flake8]
 show_source = true
 max-line-length = 88
-ignore = E402,W503
+ignore = E402,W503,E501
```

Comment:
Why should we ignore E501?

Reply:
If you convert a python library in your role to the collections format, the import statement could be longer than the original one. For example, `from ansible.module_utils.network_lsr.utils import Util` becomes 
`from ansible_collections.fedora.system_roles.plugins.module_utils.network_lsr.utils import Util`. To access an object in the collections, it's supposed to give Fully Qualified Collection Name (FQCN), which starts with ansible_collections.collection_namespace.collection_name. Plus, `module_utils` dir is moved to `plugins` in the collections format. To handle the length, we had to add E501.

Reply:
Good catch @ffmancera - @nhosoi instead of disabling the error globally, it should be disabled for the respective line with appending a comment like `# noqa: E501` to ensure that the remaining files are still checked.